### ISSUE TITLE
Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

### ISSUE THREAD

**Issue by gaetanm** [activeadmin/arbre#268]
Hello,

When I run Rspec on my project I have that warning:
> ruby/gems/2.7.0/gems/arbre-1.4.0/lib/arbre/element.rb:182: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

I guess the fix is to change
```
helpers.send(name, *args, &block)
```
to
```
helpers.send(name, *args, **kwargs, &block))
```

ActiveAdmin 2.9.0
Rails 6.0.3

**Comment by javierjulio** [comment]
I would expect our tests to fail now that we have added Ruby 3.0 CI builds (pending merge) but they aren't. I don't see warnings on earlier version builds either. We likely don't have test coverage for this case. Were you able to determine where that was occurring and if so could you share what that code looked like? Could give a hint on how to create a reproducible test case. Thanks.

**Comment by amelie-certin** [comment]
Hello, I managed to reproduce it by doing so:
```ruby
# helper
def foo(bar: 2)
  # Do something useful
end


  form do |f|
    f.input :name, hint: foo(bar: 1)
    ...
  end
```
Well, in our case we are using a helper to format a collection for a select, but it is all the same as long as the helper method takes a named parameter

**Comment by danny-lai-queo** [comment]
my server is flooded with that same warning .    Is  there any fix  ? 

I  tried gaetanm 's suggestion to use 
helpers.send(name, *args, **kwargs, &block)    # with kwargs = {}

but it will crash and does not work .

**Comment by javierjulio** [comment]
This was fixed in #318 and part of the latest release. Thanks!
